style.css

```diff
@@ -1 +1,199 @@
 /*  Add your styling here */
+@import url('https://fonts.googleapis.com/css?family=Courgette|Lato:400,700,900|Raleway|Roboto:400,700&display=swap');
+* {
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Courgette;
+    margin:0;
+    color: #50210d;
+    padding: 1px;
+    background: #F5C7DE;
+    height: auto;
+  }
+  
+  p{
+      padding: 0px 5px;
+  }
+
+  /* Header */
+  .header {
+    padding: 10px;
+    text-align: center;
+    
+  }
+  
+  .header h1 {
+    font-size: 2rem;
+    color:#825947;
+  }
+
+  h2 {
+      color:#b81c4c
+  }
+
+  main{
+      max-width: 900px;
+      margin:auto;
+      box-shadow: 30px 0px 40px rgba(0,0,0,0.4), -30px, 0px, 40px rgba(0,0,0,0.4); /*not working*/
```

Comment:
This can only be one value, so just:
```
box-shadow: 30px 0px 40px rgba(0,0,0,0.4)
```

This should then add a shadow effect against your whole main element